Polyhedron/demo/Polyhedron/Plugins/Classification/Classification_plugin.cpp

```diff
@@ -1371,10 +1367,8 @@ public Q_SLOTS:
       QSpinBox* trials = dialog.add<QSpinBox> ("Number of trials: ", "trials");
       trials->setRange (1, 99999);
       trials->setValue (500);
-      QDoubleSpinBox* rate = dialog.add<QDoubleSpinBox> ("Learning rate: ", "learning_rate");
-      rate->setRange (0.00001, 10000.0);
-      rate->setValue (0.001);
-      rate->setDecimals (5);
```

Comment:
I wonder why you change that. Given that your new class has `setRange` and `setValue`.

Reply:
At first I didn't think to code those functions, I did it when I realized there were FAR MORE QDoubleSpinBoxes than I thought and it became clear that adding those functions would be a lot faster than to remove the calls everywhere. So the first classes I worked on were managed differently.